examples/random_walk_qpe.py

```diff
@@ -68,10 +70,10 @@ def random_walk_phase_estimation(
 def example_controlled_oracle(q1: qubit, q2: qubit, t: float) -> tuple[qubit, qubit]:
```

Comment:
should this function take an angle?

Reply:
Depends how you interpret things. I always think of the `t` in `e^(-itH)` as time so I kept float here. But happy to change the angle if you prefer (will require more explicit angle to float and vice versa conversions though)

Reply:
never mind